packages/react-devtools-core/src/backend.js

```diff
@@ -60,18 +60,26 @@ function debug(methodName: string, ...args: Array<mixed>) {
   }
 }
 
+type ReactNativeProfilingSettings = {
+  recordChangeDescriptions: boolean,
+};
 export function initialize(
   maybeSettingsOrSettingsPromise?:
     | DevToolsHookSettings
     | Promise<DevToolsHookSettings>,
   shouldStartProfilingRightNow: boolean = false,
-  profilingSettings?: ProfilingSettings,
+  profilingSettings?: ReactNativeProfilingSettings,
 ) {
+  const mergedProfilingSettings: ProfilingSettings | void =
+    typeof profilingSettings === 'object' && profilingSettings != null
+      ? {...profilingSettings, recordTimeline: false}
```

Comment:
Wait so does it mean we will never respect `recordTimeline: true`  pass in from `initialize`? Could we add a comment explaining this? e.g. if `store` is the only way to set this to true

Reply:
Good catch, yeah, I should just default it to `false` here. Forgot that `react-devtools-core` might be used not only for React Native.